lib/api.js

```diff
@@ -10,7 +10,7 @@
  *
  * @namespace tree
  */
-var api = {
+module.exports = function Api () {
```

Comment:
```js
function API (tree) {
   this.tree = tree
   ...
}
```

Reply:
why is this?

Reply:
Alternative to `.call`

Reply:
It is kind of redundant and not a good solution either